MapboxCoreNavigation.podspec

```diff
@@ -43,6 +43,7 @@ Pod::Spec.new do |s|
   s.dependency "MapboxNavigationNative", ">= 101.0.1", "< 102.0.0"
   s.dependency "MapboxDirections", "~> 2.5.0"
```

Comment:
Should be bumped to 2.6.0?

Reply:
I think that Directions `v2.6.0` will be automatically picked up anyway if we select up to next minor version. If you feel that it's useful to get new stuff from Directions `v2.6.0` even for this specific version I can update it.